src/generate_value_types.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+# 'register' is a deprecated keyword in C++1x
+gperf -t value_types_by_class.gperf | sed 's/register //g' > value_types_by_class.hpp
+gperf -t value_types_by_cql.gperf | sed 's/register //g' > value_types_by_cql.hpp
```

Comment:
If a user calls the script from outside of the src dir, the generated files won't go to the src directory. How about something like this (untested):
```
DIRNAME=`dirname "$0"`
for i in value_types_by_class value_types_by_cql ; do
  gperf -t "$DIRNAME/$i.gperf" | sed 's/register //g' > "$DIRNAME/$i.hpp"
done
```
No one should be insane enough to use spaces in their directory paths, but the extra quoting above handles it if they do.